src/NUnitFramework/tests/Internal/TestExecutionContextTests.cs

```diff
@@ -256,6 +256,13 @@ public void TestCanAccessItsOwnProperties()
             Assert.That(TestExecutionContext.CurrentContext.CurrentTest.Properties.Get("Answer"), Is.EqualTo(42));
         }
 
+        [Test]
```

Comment:
Here I was not really sure where to add tests for the `Test/TestMethod/TestSuite` classes. I see there is a lot of various test files checking the behaviour of functionalities depending on such classes but I could not locate the tests for those classes themselves.

Because of that I implemented the test in place that may use those exposed Arguments property, and written the test in the same way as all other tests are written in that file.

Could you please point me to the place that would be best suited for testing this change?